sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/ResolveHintsSuite.scala

```diff
@@ -30,37 +30,48 @@ import org.apache.spark.sql.types.IntegerType
 class ResolveHintsSuite extends AnalysisTest {
   import org.apache.spark.sql.catalyst.analysis.TestRelations._
 
+  private def checkAnalysisWithTempViewEliminated(
```

Comment:
This appears twice. Shall we move it to `AnalysisTest`? And probably name it `checkAnalysisWithoutViewWrapper`